runtime/src/main/java/org/corfudb/runtime/clients/SequencerClient.java

```diff
@@ -54,9 +57,10 @@ public SequencerClient(IClientRouter router, long epoch, UUID clusterID) {
      * @param streamsAddressesRange requested streams and ranges.
      * @return streams address maps in the given range.
      */
-    public CompletableFuture<StreamsAddressResponse> getStreamsAddressSpace(List<StreamAddressRange> streamsAddressesRange) {
-        return sendMessageWithFuture(CorfuMsgType.STREAMS_ADDRESS_REQUEST.payloadMsg(
-                new StreamsAddressRequest(streamsAddressesRange)));
+    public CompletableFuture<StreamsAddressResponse> getStreamsAddressSpace(
+            List<StreamAddressRange> streamsAddressesRange) {
+        return sendRequestWithFuture(getStreamsAddressRequestMsg(streamsAddressesRange),
+                false, false);
```

Comment:
Actually, looking at the test cases in SequencerServerTest suggests that this is not an issue.

Reply:
Java doesn't have unsigned longs. When the system is first bootstrap the trim mark should be less than 0. Otherwise address 0 will be incorrectly considered trimmed. Why not use int64?

Reply:
Thanks for confirming. This change has been made.